packages/server/utils/publish.ts

```diff
@@ -1,23 +1,59 @@
 import getPubSub from './getPubSub'
-import {
-  MeetingSubscriptionPayload,
-  NotificationSubscriptionPayload,
-  OrganizationSubscriptionPayload,
-  TaskSubscriptionPayload,
-  TeamSubscriptionPayload
-} from 'parabol-client/types/graphql'
+import {NotificationSubscriptionResponse} from '~/__generated__/NotificationSubscription.graphql'
+import {OrganizationSubscriptionResponse} from '~/__generated__/OrganizationSubscription.graphql'
+import {MeetingSubscriptionResponse} from '~/__generated__/MeetingSubscription.graphql'
+import {TaskSubscriptionResponse} from '~/__generated__/TaskSubscription.graphql'
+import {TeamSubscriptionResponse} from '~/__generated__/TeamSubscription.graphql'
 
 export interface SubOptions {
   mutatorId?: string
   operationId?: string | null
 }
 
+type NotificationPayload =
+  | NotificationSubscriptionResponse['notificationSubscription']['__typename']
+  | 'AcceptTeamInvitationPayload'
+
+type OrganizationPayload =
+  | OrganizationSubscriptionResponse['organizationSubscription']['__typename']
+  | 'DowngradeToPersonalPayload'
+
+type MeetingPayload =
+  | MeetingSubscriptionResponse['meetingSubscription']['__typename']
+  | 'AutoGroupReflectionsPayload'
+  | 'DragEstimatingTaskSuccess'
+
+type TeamPayload =
```

Comment:
There are a bunch of subscription payloads that aren't being exported from Relay, particularly the Team subscription

Reply:
big picture, this is about the only valid use of the types generated by gql2ts-- the types are built from the server schema & used ONLY by the server schema.
that said, not at all sad about removing it from both server & client!
if we want, we can add it back in later, for now, i've got no problem just calling this `any` and being a little unsafe for now